### PR TITLE
[docs] LCM

### PR DESCRIPTION
**Pull request by stevhliu**
Refactors the LCM and LCM-LoRA docs together because their usage is so similar it is redundant/harder to navigate to have separate pages for them.

todo:

- [x] add LCM-LoRA

### CODE REVIEW

File: docs/source/en/using-diffusers/inference_with_lcm.md [hunk 1]
```diff
@@ -10,29 +10,30 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
 
-From the [official website](https://latent-consistency-models.github.io/):
+This guide will show you how to use LCMs and LCM-LoRAs for fast inference on tasks and how to use them with other adapters like ControlNet or T2I-Adapter.
 
-> LCMs can be distilled from any pre-trained Stable Diffusion (SD) in only 4,000 training steps (~32 A100 GPU Hours) for generating high quality 768 x 768 resolution images in 2~4 steps or even one step, significantly accelerating text-to-image generation. We employ LCM to distill the Dreamshaper-V7 version of SD in just 4,000 training iterations.
+> [!TIP]
+> LCMs and LCM-LoRAs are available for Stable Diffusion v1.5, Stable Diffusion XL, and the SSD-1B model. You can find their checkpoints on the [Latent Consistency](https://hf.co/collections/latent-consistency/latent-consistency-models-weights-654ce61a95edd6dffccef6a8) Collections.
```
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_lcm.md:24]: Very nice usage of collections here!

File: docs/source/en/using-diffusers/inference_with_lcm.md [hunk 2]
```diff
@@ -49,31 +50,69 @@ pipe = StableDiffusionXLPipeline.from_pretrained(
...
+</hfoption>
+<hfoption id="LCM-LoRA">
+
+To use LCM-LoRAs, you need to replace the scheduler with the [`LCMScheduler`] and load the LCM-LoRA weights with the [`~loaders.LoraLoaderMixin.load_lora_weights`] method. Then you can use the pipeline as usual and pass a text prompt to generate an image in just 4 steps.
+
+A couple of notes to keep in mind when using LCM-LoRAs are:
+
+* Typically, batch size is doubled inside the pipeline for classifier-free guidance. But LCM applies guidance with guidance embeddings and doesn't need to double the batch size which leads to faster inference. The downside is that negative prompts don't work with LCM because they don't have any effect on the denoising process.
+* You could use guidance with LCM-LoRAs, but it is very sensitive to high `guidance_scale` values and can lead to artifcats in the generated image. The best values we've found are between [1.0, 2.0].
+* Replace [stabilityai/stable-diffusion-xl-base-1.0](https://hf.co/stabilityai/stable-diffusion-xl-base-1.0) with any finetuned model you want. For example, try using the [animagine-xl](https://huggingface.co/Linaqruf/animagine-xl) checkpoint to generate anime images with SDXL.
```
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_lcm.md:73]: Do you think we could rewrite this section, eliminating the points that come from the points we already noted for LCM?

**Review by sayakpaul — APPROVED** [review]
Solid!

File: docs/source/en/using-diffusers/inference_with_lcm.md (hunk 2 above)
Comment (stevhliu) [reply docs/source/en/using-diffusers/inference_with_lcm.md:73]: Yeah I think its probably best to keep the LCM-LoRA notes under the `hfoption`. This way it is entirely self-contained and complete

**Review by yiyixuxu — APPROVED** [review]
great :)

**Pull request merged**